### ISSUE TITLE
‌ When there is a lot of contextual content in a conversation, incomplete responses may occur.

### ISSUE THREAD

**Issue by joeaki1983** [cg-dot/vertexai-cf-workers#11]
![image](https://github.com/user-attachments/assets/bc16082e-3630-4dc0-a223-19f8857a92a0)
![image](https://github.com/user-attachments/assets/38d1aa55-7e39-48b6-b80b-e3460c3dbf59)

 As shown in the image, when there is a lot of context in a conversation, incomplete responses often occur. How can this issue be resolved?

**Comment by cg-dot** [comment]
I haven't been able to reproduce this issue on my end. It could be kind of network issue on your end. You might be able to find more details in your logs to further investigate the cause of this issue.

**Comment by joeaki1983** [comment]
> I haven't been able to reproduce this issue on my end. It could be kind of network issue on your end. You might be able to find more details in your logs to further investigate the cause of this issue.

 Logs? Are you referring to the client-side logs?

**Comment by cg-dot** [comment]
> Logs? Are you referring to the client-side logs?

Yes, there might be details in the logs of the client you're using.

**Comment by joeaki1983** [comment]
> > Logs? Are you referring to the client-side logs?
> 
> Yes, there might be details in the logs of the client you're using.

 I figured it out. It's because Cloudflare limits the performance for free users, and my usage intensity was too high.
